client/blocks/product-selector/index.jsx

```diff
@@ -177,10 +177,16 @@ export class ProductSelector extends Component {
 			return null;
 		}
 
+		const subscribedMoment = purchase.subscribedDate ? moment( purchase.subscribedDate ) : null;
+
+		const expiryMoment = purchase.expiryDate
```

Comment:
Just confirming that this is a good place for those `moment()` calls - the `ProductSelector` acts like a controller component.